### DISCUSSION TITLE
Images couldn't been show properly on github.io

### DISCUSSION THREAD

**Discussion by peteryuhang** [cotes2020/jekyll-theme-chirpy#1323] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I write the post with the image set below

```
![](/assets/images/%E4%BA%8C%E9%A1%B9%E5%A0%86_%E9%93%BE%E8%A1%A8%E8%A1%A8%E7%A4%BA.png){: width="372" height="89" }
```

Previously I can see the image on web, but currently all the the image seems disappear, and it show sth below:

<img width="772" alt="image" src="https://github.com/cotes2020/jekyll-theme-chirpy/assets/32042176/7b1349ca-31f9-4462-9e65-0aacaafca7bb">

Not sure if there is something I missed. Thanks!

The github.io website: https://peteryuhang.github.io/

The repo: https://github.com/peteryuhang/peteryuhang.github.io

### Environment

Ruby: ruby 3.2.2 (2023-03-30 revision e51014f9c0) [x86_64-darwin21]
Jekyll: jekyll 4.3.2
Chirpy: jekyll-theme-chirpy (6.2.3)

### Operations you have already tried

On my local, tested by using `bundle exec jekyll s`, the result looks OK. The problem only appear on production

**Comment by NichtsHsu** [comment]
我看了一下，你 fork 的 chirpy-starter 但是拷贝了很多 jekyll-theme-chirpy 的 js/html 到你的仓库，这是很不好的，因为一旦这些文件有更新，你那边是完全获取不到的。我估计你这个问题也是跟原始仓库文件分离导致的。

**Reply by peteryuhang** [reply]
感谢提醒，想请教一下，如何才能跟原始仓库文件保持同步呢？我这个 repo 是从 chirpy-starter 生成的，但是之前出现了很多显示上面的问题，所以我就拷贝了 jekyll-theme-chirpy 的一些文件，我需要重新创建一个 repo 基于 jekyll-theme-chirpy 吗？多谢🙏

**Reply by NichtsHsu** [reply, marked as answer]
如果需要深度客制化，最好是 fork jekyll-theme-chirpy，然后将 jekyll-theme-chirpy 设置为本地仓库的 upstream，利用 git pull upstream master 同步新修改（可能需要解决合并冲突）。

**Reply by peteryuhang** [reply]
好的，我去试一试
